### PR TITLE
Mutual information fixes

### PR DESCRIPTION
**Pull request by scottwittenburg**

### CODE REVIEW

**Comment by scottwittenburg** [comment]
I think this is just about ready.  I believe we now have all the functionality working as before the refactor from component-experiments.

@jourdain Mind taking a look?  Feel free to stop by and see it live if you want.

File: src/InfoViz/Native/ParallelCoordinates/index.js [hunk 1]
```diff
@@ -294,6 +297,7 @@ function parallelCoordinate(publicAPI, model) {
           .attr('width', glyphSize)
           .attr('height', glyphSize)
           .style('color', d.legend.color) // Firefox SVG use color bug workaround fix
+          .classed(style.clickable, (data, idx) => d.annotated)
```
Comment (aronhelser) [inline src/InfoViz/Native/ParallelCoordinates/index.js]: I don't think this is doing what you want? (data) as an arg, (d.annotated) in the body?

File: src/InfoViz/Native/ParallelCoordinates/index.js [hunk 2]
```diff
@@ -881,6 +825,11 @@ const DEFAULT_VALUES = {
 
   selectionOpacityAdjustment: 1,
   polygonOpacityAdjustment: 1,
+
+  hoverIndicatorHeight: 10,
+  hoverIndicatorWidth: 7,
+
+  numberOfBins: 128,
```
Comment (aronhelser) [inline src/InfoViz/Native/ParallelCoordinates/index.js]: Is this just a default?

File: src/InfoViz/Native/ParallelCoordinates/index.js (hunk 1 above)
Comment (jourdain) [inline src/InfoViz/Native/ParallelCoordinates/index.js]: Seems to be a typo. Good catch @aronhelser

**Comment by jourdain** [comment]
LGTM with the comment that Aron made about the d.annotation

File: src/InfoViz/Native/ParallelCoordinates/index.js (hunk 1 above)
Comment (scottwittenburg) [inline src/InfoViz/Native/ParallelCoordinates/index.js]: It's doing what I want, just confusing to readers.  I'll get rid of `(data, idx)`, as what I want is `d.annotated`.

File: src/InfoViz/Native/ParallelCoordinates/index.js (hunk 2 above)
Comment (scottwittenburg) [inline src/InfoViz/Native/ParallelCoordinates/index.js]: Yes, I set it to the max as a default, but it's set by asking the histogram 2d provider for the number of bins elsewhere.

File: src/InfoViz/Native/ParallelCoordinates/index.js (hunk 1 above)
Comment (aronhelser) [inline src/InfoViz/Native/ParallelCoordinates/index.js]: OK, sounds good. +1

**Pull request merged**